payloads/external/edk2/Kconfig

```diff
@@ -311,6 +311,13 @@ config EDK2_DASHARO_POWER_CONFIG_FAN_CURVE
 
 endif
 
+config EDK2_DASHARO_USB_CONFIG
```

Comment:
ALreayd present in line https://github.com/Dasharo/coreboot/blob/2344a7c8fecc38bfdecd60f05727d463c8bcfad2/payloads/external/edk2/Kconfig#L278

Reply:
as in comment above